pmm-tests/backup/ps_80_bm.yml

```diff
@@ -159,3 +159,34 @@
       name: qpress
       state: present
     when: ansible_os_family == "RedHat"
+
+  - name: Kill running pmm-agents
+    ignore_errors: no
+    shell: killall pmm-agent
+
+  - name: "add pmm-client binary to the secure path"
+    become: yes
+    become_user: root
+    lineinfile:
+      path: /etc/sudoers
+      # The String to Search
+      regexp: "^Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin$" 
+      # The String to Replace
+      line: "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/home/ec2-user/workspace/aws-staging-start/pmm2-client/config:/usr/local/percona/pmm2/bin:/usr/local/percona/pmm2/config"
+      state: present
+      backup: yes
+      validate: '/usr/sbin/visudo -cf %s'
+
+  - name: create config file
+    ignore_errors: yes
+    shell: pmm-agent setup --config-file=/home/ec2-user/workspace/aws-staging-start/pmm2-client/config/pmm-agent.yaml --server-address=localhost:443 --server-insecure-tls --server-username=admin --server-password=admin --force
+    become: yes
+    become_user: root
+
+  - name: run pmm-agent as root user
+    ignore_errors: yes
+    shell: pmm-agent --config-file=/home/ec2-user/workspace/aws-staging-start/pmm2-client/config/pmm-agent.yaml >> pmm-agent_log.txt
+    async: 2592000
+    poll: 0
+    become: yes
+    become_user: root
```

Comment:
add empty last line

Reply:
@VoK0 my only concern with this change is, it makes it difficult for anyone to setup locally, those paths are meant to work only on staging, can we try to find an alternative way?